runtime/include/sandbox_functions.h

```diff
@@ -215,6 +215,16 @@ sandbox_mem_print_perf(struct sandbox *sandbox)
         uint32_t blocked_us      = sandbox->blocked_duration / runtime_processor_speed_MHz;
         uint32_t returned_us     = sandbox->returned_duration / runtime_processor_speed_MHz;
 
+	if (sandbox->module->next_module == NULL) {
+                uint32_t now = __getcycles();
+                bool miss_deadline = now > sandbox->absolute_deadline ? true : false;
```

Comment:
I may be missing something here (like when this will be called), but in any case I think instead of "now" using sandbox's `completion_timestamp` would be more accurate.

Reply:
You are right, thanks for catching this potential bug. This is for evaluating the performance of the scheduling algrithms, not related to any functionalities of sledge.